src/store.cc

```diff
@@ -540,7 +540,17 @@ storeGetPublic(const char *uri, const HttpRequestMethod& method)
 StoreEntry *
 storeGetPublicByRequestMethod(HttpRequest * req, const HttpRequestMethod& method, const KeyScope keyScope)
```

Comment:
There is also another `storeGetPublic()` for getting public entries. I assume we should not care about it since it has no 'request' in its context and, therefore is not a subject for collapsing requests.